firmware/README.md

```diff
@@ -0,0 +1,531 @@
+# BraveSensor-Firmware
+
+## Table of Contents
```

Comment:
I like Heidi's table of contents (or whoever made this). We should eventually do this in the others too.